launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -540,3 +593,17 @@ void CSettingsView::on_spinBoxReservedArea_valueChanged(int arg1)
 	node->Float() = float(arg1) / 100; // percentage -> ratio
 }
 
+void CSettingsView::addItemsToControls()
+{
+	ui->comboBoxAutosaveMode->addItem("Off");
+	ui->comboBoxAutosaveMode->setItemData(0, "Off", Qt::UserRole);
+
+	ui->comboBoxAutosaveMode->addItem("Using Counter");
+	ui->comboBoxAutosaveMode->setItemData(1, "Using Counter", Qt::UserRole);
+
+	ui->comboBoxAutosaveMode->addItem("Using Turn Info");
+	ui->comboBoxAutosaveMode->setItemData(2, "Using Turn Info", Qt::UserRole);
+
+	ui->comboBoxAutosaveMode->addItem("Using Schema");
```

Comment:
Using schema -> using pattern

Reply:
Renamed to Using Template. Seems more understandable?